sql/codegen_alps.go

```diff
@@ -43,6 +43,7 @@ type alpsFiller struct {
 	ModelDir           string
 	ScratchDir         string
 	PredictOutputTable string
+	PredictInputModel  string
```

Comment:
This field is used for https://github.com/sql-machine-learning/sqlflow/blob/develop/sql/template_alps.go#L147